modules/Kconfig

```diff
@@ -28,6 +28,7 @@ source "modules/Kconfig.mcux"
 source "modules/Kconfig.microchip"
 source "modules/Kconfig.nuvoton"
 source "modules/Kconfig.open-amp"
+source "modules/Kconfig.s32"
```

Comment:
rename to `Kconfig.nxp_s32`

Reply:
I have understood we agreed to address this in a separate PR and leave it as it is for now, please see https://github.com/zephyrproject-rtos/zephyr/pull/49161#discussion_r948042301

Reply:
@galak, After the 3.2 release, I plan on merging the various "NXP" Kconfig modules into one Kconfig.nxp to address this.